ExternalScripts(DONOTINCLUDEINRELEASE)/scraper/quest/quest_spider.py

```diff
@@ -17,14 +18,23 @@ def __init__(self) -> None:
         self.start_urls = [self.base_url_classic.format(quest_id) for quest_id in QUEST_IDS]
 
     def parse(self, response):
+        if response.url.startswith('https://www.wowhead.com/classic/quests?notFound='):
+            questID = re.search(r'https://www.wowhead.com/classic/quests\?notFound=(\d+)', response.url).group(1)
+            logging.warning('\x1b[31;20mQuest with ID {questID} not found\x1b[0m'.format(questID=questID))
+            return None
```

Comment:
Added this check to see when quests are not found, instead of silently ignoring it.

Reply:
Great idea 👍🏻